src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsresettlementpassportapi/service/POPUserMetricsService.kt

```diff
@@ -1,30 +1,98 @@
 package uk.gov.justice.digital.hmpps.hmppsresettlementpassportapi.service
 
 import io.micrometer.core.instrument.MeterRegistry
+import io.micrometer.core.instrument.Tags
 import org.slf4j.LoggerFactory
 import org.springframework.stereotype.Service
+import uk.gov.justice.digital.hmpps.hmppsresettlementpassportapi.config.ResourceNotFoundException
+import uk.gov.justice.digital.hmpps.hmppsresettlementpassportapi.data.LicenceTag
+import uk.gov.justice.digital.hmpps.hmppsresettlementpassportapi.data.PopUserCountMetric
+import uk.gov.justice.digital.hmpps.hmppsresettlementpassportapi.data.PopUserCountMetrics
+import uk.gov.justice.digital.hmpps.hmppsresettlementpassportapi.service.external.PrisonRegisterApiService
 
 @Service
 class POPUserMetricsService(
   private val registry: MeterRegistry,
-  private val prisonOTPService: PoPUserOTPService,
+  private val popUserOTPService: PoPUserOTPService,
```

Comment:
Can you add a test for this service please?

Reply:
test included now